variants/metal-k8s-1.27/Cargo.toml

```diff
@@ -16,6 +16,7 @@ partition-plan = "unified"
 [package.metadata.build-variant.image-features]
 grub-set-private-var = true
 unified-cgroup-hierarchy = true
+systemd-networkd = true
```

Comment:
why is this present in only this one variant?

Reply:
good catch i've removed it